fairlearn/linear_model/__init__.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""This module contains existing linear models that have been adapted to work with fairness constraints."""
+
+from ._fair_logistic import FairLogisticRegression
+
+__all__ = [
+    "FairLogisticRegression"
```

Comment:
For the naming of the class, please use something similar to: https://github.com/fairlearn/fairlearn/issues/466#issuecomment-650564824

Instead of `FairLogisticRegression`, I would suggest to go with `ConstrainedLogisticRegression`, and then represent the specific type of fairness constraints using a parameter.

Reply:
Thanks! I've changed this now. 
I've also implemented a way to specify a `covariance_bound` per sensitive feature, but I am not really satisfied with how this is happening right now.
The class gets instantiated as `ConstrainedLogisticRegression(constraints="demographic_parity", covariance_bound=[0.0, 10.0]` which indicates that the first sensitive feature has a threshold of 0, and the second has a threshold of 10. However, the sensitive features are only provided in the `fit` function, thus not when instantiating the classifier (e.g. `clf.fit(X, y, sensitive_feature_ids=[2, 3])`. 

What would be the correct way to go about this? I think it is valuable to be able to provide a threshold per sensitive feature, as it allows the user to have a finer balance over fairness and accuracy.